Src/library/wel/windows/wel_window.e

```diff
@@ -2774,6 +2786,24 @@ feature {WEL_INPUT_EVENT} -- Externals
 			"InvalidateRect"
 		end
 
+feature -- Externals DPI
+
+	cwin_get_dpi_for_window (hwnd: POINTER): INTEGER
```

Comment:
Please add comments (maybe from the C api), for those who don't know about this api.

Reply:
Done.